app/controllers/entity_controller.rb

```diff
@@ -0,0 +1,34 @@
+class EntityController < ApplicationController
+  def index
+    @category = Category.find(params[:category_id])
+    @entities = Entity.where(category_id: params[:category_id], user: current_user).order('created_at ASC')
+  end
```

Comment:
- [ ] [Optional] It would be nice to have a method for arranging the order in the model file. Remember that the logic in the controller should be lean. 👍